plugins/quota/actor.go

```diff
@@ -0,0 +1,100 @@
+package quota
+
+import (
+	"os"
+	"path"
+
+	"github.com/gluster/glusterd2/glusterd2/daemon"
+	"github.com/gluster/glusterd2/glusterd2/volume"
+	"github.com/gluster/glusterd2/glusterd2/xlator"
+	"github.com/gluster/glusterd2/pkg/errors"
+
+	log "github.com/sirupsen/logrus"
+)
+
+type quotadActor struct{}
+
+const (
+	quotaDaemonKey = "enable"
+)
+
+// isQuotadAffectedNode returns true if there are local bricks of volume on
+// which quota is enabled
+func isQuotadAffectedNode(volumes []*volume.Volinfo) bool {
+	for _, v := range volumes {
+		if !isQuotaEnabled(v) {
+			continue
+		} else if v.State != volume.VolStarted {
+			continue
+		} else {
+			bricks := v.GetLocalBricks()
+			if len(bricks) > 0 {
+				return true
+			}
+			continue
```

Comment:
I don't think continue is required here, can you please check

Reply:
yep. its not required.